app/views/load_sample.py

```diff
@@ -26,6 +27,9 @@ def upload_sample(action_plan_id):
 
     # For convenience use the action plan ID as the collection exercise ID
     load_sample(sample_file, collection_exercise_id=action_plan_id,
-                action_plan_id=action_plan_id)
+                action_plan_id=action_plan_id, host=Config.RABBITMQ_HOST, port=Config.RABBITMQ_PORT,
```

Comment:
Use the app's config object here instead? Something like:
[Import](https://github.com/ONSdigital/census-rm-ops/blob/master/app/controllers/action_controller.py#L2)
[Usage](https://github.com/ONSdigital/census-rm-ops/blob/master/app/controllers/action_controller.py#L9)

Reply:
The `app.config` pattern is actually recommended here as this is all (presumably) at "request time".

Reply:
done